tests/css_check_test.js

```diff
@@ -97,16 +108,17 @@ Scenario('check for css values', async ({ I }) => {
     //check if css from assets corresponds to what we see on page
     I.assertContain(cssAttr, convert.keyword.rgb(css.value).join(', ').toString());
     
-    await I.executeScript(async () => {
-        eval(await evolv.client.getActiveKeys()).current.forEach(element => {
-            //get list of active keys
-            console.log(`KEY:${element}`);
+    let activeKeys = await I.executeScript(async () => {
```

Comment:
My opinion here for how we should test these is use the exact keys we expect to be present -- not by referencing the asset.js file.

The reason for this is that we are going to have multiple experiments and contexts present in the assets.js file that are not active keys.

I think we should commit the yml file we are testing against into this repo, then explicitly check the active keys match the strings we expect

Reply:
@robertsevernsentient I'm not following your proposal to include the YML file (I presume that means the metamodel file) and to check active keys against that 🤔

Reply:
I've hardcoded active key that being used in this experiment for now. Also added YML file. But we need to decide on approach how to make this flexible to avoid hardcoding activekeys.
Maybe I can parse YML file for keys that are being used on page or smthng like this. 
I'd like to hear your suggestion on how to get keys to check with  `getActiveKeys()`